src/coq/representationutils/defutils.ml

```diff
@@ -56,11 +56,11 @@ let edeclare ident (_, poly, _ as k) ~opaque sigma udecl body tyopt imps hook re
   let univs = Evd.check_univ_decl ~poly sigma udecl in
   let ubinders = Evd.universe_binders sigma in
   let ce = Declare.definition_entry ?types:tyopt ~univs body in
-  DeclareDef.declare_definition ident k ce ubinders imps hook
+  DeclareDef.declare_definition ~ontop:None ~hook_data:hook ident k ce ubinders imps
 
 (* Define a new Coq term *)
 let define_term ?typ (n : Id.t) (evm : evar_map) (trm : types) (refresh : bool) =
-  let k = (Global, Flags.is_universe_polymorphism(), Definition) in
+  let k = (Global, Attributes.is_universe_polymorphism(), Definition) in
```

Comment:
Not sure calling is_universe_polymorphism directly is a good idea, usually you are supposed to get it from the command using the attribute.